etc/7z.profile

```diff
@@ -1,23 +1,22 @@
+# Firejail profile for 7z
+# This file is overwritten after every install/update
 quiet
-# Persistent global definitions go here
+# Persistent global definitions
 include /etc/firejail/globals.local
-
-# This file is overwritten during software install.
-# Persistent customizations should go in a .local file.
+# Persistent local customizations
 include /etc/firejail/7z.local
 
-# 7zip crompression tool profile
-ignore noroot
-
-include /etc/firejail/default.profile
-
 blacklist /tmp/.X11-unix
 
-tracelog
+ignore noroot
```

Comment:
Note to self: check if `ignore` has to go before `include /etc/firejail/default.profile`